country_limiter.go

```diff
@@ -13,8 +14,9 @@ import (
 )
 
 type CountryLimiter struct {
-	db        *maxminddb.Reader
-	countries []string
+	db            *maxminddb.Reader
+	countries     map[string]bool
+	skipCountries map[string]bool
```

Comment:
Using `struct{}` is memory efficient.

```suggestion
	countries     map[string]struct{}
	skipCountries map[string]struct{}
```

ref: https://speakerdeck.com/izzii/hong-tuteli-jie-suru-go-konpairazui-shi-hua-go-conference-mini-2023?slide=12

Reply:
便利っぽい